src/components/Rockets.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import RocketCard from './RocketCard';
+
+const Rockets = () => {
+  const state = useSelector((state) => state.storeSlice);
+  // const dispatch = useDispatch();
+
+  // useEffect(() => {
+  //   dispatch(loadRockets());
+  // }, []);
```

Comment:
- [ ] Please remove commented code as it will make the code cleaner :sparkles:

Reply:
@karlavdelgadof  yes you are right i will change it